### ISSUE TITLE
OpenSearch Dashboards: wrong check for `startupProbe`

### ISSUE THREAD

**Issue by rursprung** [opensearch-project/helm-charts#389]
the `startupProbe` in the helm chart is only added if `readinessProbe` is defined, this is probably a simple copy&paste mistake:
https://github.com/opensearch-project/helm-charts/blob/72078b10070c44e560f78119d79b3930e2013b79/charts/opensearch-dashboards/templates/deployment.yaml#L105-L110

for reference: for OpenSearch the content is just always added:
https://github.com/opensearch-project/helm-charts/blob/72078b10070c44e560f78119d79b3930e2013b79/charts/opensearch/templates/statefulset.yaml#L335-L338

**Comment by TheAlgo** [comment]
@rursprung do you intend to pick up this issue?

**Comment by rursprung** [comment]
> @rursprung do you intend to pick up this issue?

no. since i'm not using the helm charts (i just looked at them to see how you did some things here) i couldn't test the change, so i'd prefer not to change things here
